### PR TITLE
Proof-read of the readme

### PR DESCRIPTION
**Pull request by xeviknal**
A little bit of this, a little bit of that.
Spell-check and some small proposal changes. nothing big.
If I have time tomorrow, I'll propose another PR

### CODE REVIEW

File: README.md
Comment (xeviknal) [inline README.md:102]: Shouldn't we add a more scientific sentence for the Adam? hahahahaha
Like adding a reference to theory slides where it compares both of them. Or something like that?

**Review by ziritrion — APPROVED** [review]
LGTM. Want me to wait for you to finish your review before merging?

File: README.md
Comment (ziritrion) [reply README.md:102]: I honestly can't think up of anything. Optimizers aren't the focus of this project, so I would just let it be.

File: README.md
Comment (xeviknal) [inline README.md:87]: Shouldn't we add a video with good results in the video right above?

**Review by ziritrion — APPROVED** [review]

**Pull request merged**
